lab0/INSTRUCTIONS.md

```diff
@@ -1,18 +1,20 @@
 Lab 0: Building your first Nerdpack
 ===================================
 
-The purpose of this lab is to provide hands on experience building on top of the New Relic One platform. Following the instructions in this lab, you will be up and running quickly. First, let's make sure you understand a few key concepts that you'll be working with.
+The purpose of this lab is to provide you a quick, hands on experience building on top of the New Relic One platform. First, let's make sure you understand a few key concepts that you'll be working with.
 * **Launcher**: An entry point into the New Relic One platform that is loaded on the "Apps" page.
-* **Nerdlet**: A visual component that renders inside the New Relic One platform at a well defined extension point. For example, a Launcher can invoke a Nerdlet (as you will see shortly). At their core, Nerdlets are React.js components.
-* **Nerdpack** (aka Package): A collection of artifacts, e.g. Nerdlets, that are "packaged" up and published into the New Relic One platform.
-* **Visualization:** This is another Nerdpack artifact that renders your data in a customized way. Unlike Nerdlets, you can add visualizations to dashboards. For now, you'll focus on Nerdlets, but you'll learn about visualizations in a later lab.
+* **Nerdlet**: A visual component that renders inside the New Relic One platform at a well defined extension point. For example, a launcher can invoke a Nerdlet (as you will see shortly). At their core, Nerdlets are React.js components.
+* **Nerdpack** (aka Package): A collection of Nerdpack items, e.g. Nerdlets, that are "packaged" up and published into the New Relic One platform.
```

Comment:
We should get rid of (aka Package) for a couple reasons:

1. "Package" is not interchangeable with "Nerdpack"
2. aka is an abbreviation, and we try to stay away from those when possible

A few more notes:

- We should keep our **bolding** consistent. In this case, capture the colon (:) inside the bold tags.
- Only use quotes around actual quotes or terms we're referring back to. The quotes around "packaged" are for emphasis, so let's get rid of them.
- Refer to New Relic One OR the New Relic platform. Not the New Relic One platform.
- e.g. is another abbreviation and Nerdpack items is a recursive definition. Let's just list the items since there are only three.

References:

- [New Relic One platform](https://docs.google.com/document/d/1cPV1hcNa5QQgyPNPJbIVL-1NwtP5fqoJubcs33xHKeI/edit)

Recommended change:

> * **Nerdpack:** A collection of Nerdlets, launchers, and visualizations that are packaged up and published to the New Relic platform.